packages/convertkit/src/api/subscribe.ts

```diff
@@ -11,7 +11,7 @@ const subscribe = async (req: NextApiRequest, res: NextApiResponse) => {
   if (req.method === 'POST') {
     try {
       const {
-        email_address,
+        email,
```

Comment:
convertkit refers to this specifically as `email` for this use case and other subscribe actions and we were transforming it away so I'm changing it back. I don't know for sure why we reverted to the `email_address` format so if this doesn't work let me know

Reply:
the reason is that when we were using default form action, the email field had to be named `email_address` for it to work properly. but we can change it back to `email` with this approach I think.